torchao/prototype/moe_training/scaled_grouped_mm.py

```diff
@@ -291,13 +291,13 @@ def forward(
         ctx.out_dtype = out_dtype
         ctx.emulated = emulated
 
-        # A_mx shape: (M, K)
+        # A_fp8 shape: (M, K)
```

Comment:
I'm not a fan of `A_f8`, this isn't going to scale to other dtypes.  `A_data` is IMO better and more representative of the MX format being composed of raw data and scale.

Reply:
Alright I went ahead and changed _fp8 suffix to _data everywhere